### PR TITLE
PD-232   Rework Network Port ID Articles: R-Series

### PR DESCRIPTION
**Pull request by tonyriv3**

### CODE REVIEW

File: content/Hardware/RSeries/RSeriesNetworkPortIDs.md [hunk 1]
```diff
@@ -5,17 +5,222 @@ weight: 40
...
+TTrueNAS identifies the network ports on TrueNAS R-Series systems based on what software is installed.
 
 {{< include file="/_includes/NetworkPortIdentifications.md" type="page" >}}
 
 By default, a TrueNAS R-Series with CORE installed identifies its network ports with **ixl** followed by a number.
 
 SCALE identifies its network ports with **eno** followed by a number.
 
-The tables below list the default identification for R-Series systems and any add-on networking cards that have been qualified for use with an R-Series system.
+The tables within the expandable boxes below list the default identification for R-Series systems and any add-on networking cards that iXsystems has qualified for use with R-Series systems.
```
Comment (micjohnson777) [inline content/Hardware/RSeries/RSeriesNetworkPortIDs.md:16]: hum... maybe this?
```suggestion
The expandable boxes below contain tables that list the default identification for R-Series systems and any add-on networking cards qualified by iXsystems for use with R-Series systems.
```

File: content/Hardware/RSeries/RSeriesNetworkPortIDs.md [hunk 2]
```diff
@@ -5,17 +5,222 @@ weight: 40
...
 
-## R10 / R20 / R20A / R40 / R50
+## R10
+
```
Comment (micjohnson777) [inline content/Hardware/RSeries/RSeriesNetworkPortIDs.md:19]: Why are you using the code emphasis for the port values? if not in a or as command, using the ` ` tags is not appropriate emphasis. You could add a column to these tables for Port number, then the value in another column but still not with the command emphasis.

**Review by micjohnson777 — COMMENTED** [review]
On the comment about the command emphasis, that applies to all the port values in each table. there is no need for that emphasis as these are not commands or documented as commands. The values entered are not commands or talking about command syntax.

File: content/Hardware/RSeries/RSeriesNetworkPortIDs.md (hunk 1 above)
Comment (tonyriv3) [reply content/Hardware/RSeries/RSeriesNetworkPortIDs.md:16]: That sounds good.

**Review by micjohnson777 — APPROVED** [review]

**Pull request merged**